rosetta-source/src/main/resources/result-json-files/mappers/example-6/example-6-1.json

```diff
@@ -11,15 +11,5 @@
         } ]
       }
     }
-  },
-  "vehicle" : {
-    "engineSpecification" : {
-      "resolvedAmount" : {
```

Comment:
did you find out why this is being removed

Reply:
yes - there was a fix in translate

Reply:
in version 11.17.1 https://github.com/REGnosys/rosetta-components/tree/11.17.1